test/suite/test_config12.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# test_config12.py
+#   Test the debug configuration setting for config validation.
+#
+
+import re
+import wttest
+from contextlib import contextmanager
+
+class test_config12(wttest.WiredTigerTestCase):
+
+    @contextmanager
+    def expect_verbose(self, config, patterns, expect_output = True):
```

Comment:
Perhaps we can define this function inside `WiredTigerTestCase` instead of having two definitions here and in `test_verbose01.py`. 
@etienneptl you wrote the original `test_verbose01.py`. Was there a reason we defined expect_verbose in `test_verbose01.py`?

Reply:
Simply because it was the only test using that function and it was designed to meet the needs of `verbose01`. This version is slightly different than the one defined in `verbose01` but I agree that it should be accessible from any test, it can be useful.